ml-agents/mlagents/trainers/tensorflow_to_barracuda.py

```diff
@@ -1499,23 +1499,13 @@ def very_slow_but_stable_topological_sort(nodes, verbose):
 #########################################################
 
 
-def convert(
-    source_file,
+def convert_graph_def(
+    source_graph_def,
     target_file,
     trim_unused_by_output="",
     verbose=False,
     compress_f16=False,
 ):
```

Comment:
Removed docstring but did not replace it with something better

Reply:
I didn't remove this docstring actually -- it still lives with the `convert` function.  Since `convert` is part of the end-user interface to `tensorflow_to_barracuda.py` I thought it made more sense for the docstring to live there.  We could also have a docstring for this function, it will just duplicate a lot of the content.